content/en/blog/2024/docs-localized.md

```diff
@@ -0,0 +1,42 @@
+---
+title: OpenTelemetry website goes multilingual!
+linkTitle: Multilingual website
+date: 2024-08-21
+author: >-
+  [Severin Neumann](https://github.com/svrnm) (Cisco),
+issue: 4863
+sig: Comms
+cSpell:ignore: Comms
+---
+
+We are happy to announce that the OpenTelemetry documentation now supports
+multiple languages! Teams of contributors have already started to translate our
+resources into Portuguese (pt), Chinese (zh), and Japanese (ja).
+
+The OpenTelemetry project has grown to include contributors and users from all
+over the world. Making our documentation available in multiple languages is an
+important step for ensuring that everyone, regardless of their native language,
+can contribute to the project. We are also thrilled that this initiative gives
+end users access to multilingual documentation, making it easier to learn and
+understand OpenTelemetry.
+
+To access the documentation in your preferred language, use the language
+selector at the top right of the page.
+
+This is just the beginning! A team of contributors has just begun to translate
+the documentation into Spanish (es), and we expect to add more languages in the
+future.
```

Comment:
```suggestion
More localizations are underway, including Spanish, and we expect supporting more languages in the
future.
```

Reply:
since we have merged Spanish now, we should have it with the other ones I guess